airbyte-integrations/connectors/source-chartmogul/source_chartmogul/spec.yaml

```diff
@@ -6,7 +6,6 @@ connectionSpecification:
   required:
     - api_key
     - start_date
-    - interval
```

Comment:
look like braking change 

What occurs if a user already has an established connection with a set interval?

Reply:
Yes, It's a breaking change.